plugins/BEdita/Core/config/Migrations/20230327104919_ChangeObjectsDescriptionBodySize.php

```diff
@@ -0,0 +1,55 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractMigration;
+use Phinx\Db\Adapter\MysqlAdapter;
+
+class ChangeObjectsDescriptionBodySize extends AbstractMigration
+{
+    /**
+     * Up Method.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $limit = null;
+        if ($this->getAdapter()->getAdapterType() === 'mysql') {
+            $limit = MysqlAdapter::TEXT_MEDIUM;
+        }
```

Comment:
I know this "well-known" field length is specific to MySQL, but I can't see why it shouldn't work for other engines too. Couldn't we just use the same length for all?